reana_server/rest/info.py

```diff
@@ -108,6 +108,7 @@ def info(user, **kwargs):  # noqa
               }
     """
     try:
+        ui_config = REANAConfig.load("ui")
```

Comment:
Have you considered injecting inactivity period value through [Helm templates](https://github.com/reanahub/reana/blob/master/helm/reana/templates/reana-server.yaml#L87) similarly as it is done for `REANA_KUBERNETES_JOBS_MAX_USER_TIMEOUT_LIMIT`?. Now it's a bit strange to get it from the UI configmap